src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/StegService.kt

```diff
@@ -188,11 +188,9 @@ class StegService(
         saksbehandlerIdent: String,
         behandlingSteg: BehandlingSteg<T>
     ) {
+        utførBehandlingsvalidering(behandlingSteg, saksbehandling)
```

Comment:
Hadde det vært ennå tydligere for saksbehandler hvis man i `validerGyldigTilstand` sjekket for 
```kotlin
brukerfeilHvis (stegType != BESLUTTE_VEDTAK && saksbehandling.steg.kommerEtter(BESLUTTE_VEDTAK)) {
            "Behandlinger er allerede besluttet"
        }
```

plus flyttet tilgangssjekken til etter `validerGyldigTilstand` ?